### PR TITLE
Polygon support for visualizer messenger

### LINKED ISSUE #258: Add polygon shape support for VisualizerMessenger
Sending polygons from Ai to visualizer is currently not supported. Implement the `drawPoly` method in `Util::VisualizerMessenger` class to enable sending shape messages for polygons.

### Acceptance Criteria
- [ ] C++ code can call the VisualizerMessenger drawPoly method and send a vector of vertices
- [ ] VisualizerMessenger constructs the shape message for the layer.

### PR DESCRIPTION
**Pull request by FSXAC**
### Description

Added ability to use VisualizerMessenger to send polygon shape messages

### Testing Done

Compiled; low risk

### Resolved Issues

Resolves #258 

### Length Justification

N/A

### Review Checklist

**_It is the reviewers responsibility to also make sure every item here has been covered_**

- [x] **Start of document comments**: each `.cpp` and `.h` file should have a comment at the start of it. See files in the `thunderbots/software/geom` folder for examples.
- [x] **Function comments**: All function definitions (usually in the `.h` file) should have a javadoc style comment at the start of them. For examples, see the classes defined in `thunderbots/software/geom`
- [x] **Remove all commented out code**
- [x] **Remove extra print statements**: for example, those just used for testing
- [x] **Resolve all TODO's**: All `TODO` (or similar) statements should either be completed or associated with a github issue
- [x] **Justify drops in code coverage**: If this PR results in a non-trivial drop in code coverage (a bot should post a coverage diagram as a comment), please justify why we can't test the code that's not covered.

### CODE REVIEW

**Review by garethellis0 — CHANGES_REQUESTED** [review]
This is looking good, just two small comments.

File: src/thunderbots/software/util/visualizer_messenger/visualizer_messenger.h [hunk 1]
```diff
@@ -132,8 +132,19 @@ namespace Util
...
+         * Reuqest a message to draw a polygon shape. The origin is the
+         * first vertex passed in.
+         *
+         * @param layer: The layer name this shape is being drawn to
+         * @param vertices: A vector of pairs of vertex positions
+         * @param draw_style: the drawing style of the shape
+         * @param draw_transform: the transformation of the shape
+         */
+        void drawPoly(const std::string& layer,
+                      std::vector<std::pair<double, double>>& vertices,
```
[the remark this answers is not in the document]
Comment (FSXAC) [reply src/thunderbots/software/util/visualizer_messenger/visualizer_messenger.h]: Right, will fix that today

File: src/thunderbots/software/util/visualizer_messenger/visualizer_messenger.h [hunk 2]
```diff
@@ -132,8 +132,19 @@ namespace Util
                       DrawStyle draw_style         = DrawStyle(),
                       DrawTransform draw_transform = DrawTransform());
 
-        // TODO: #258 polygon support
-        void drawPoly();
+        /**
+         * Reuqest a message to draw a polygon shape. The origin is the
```
[the remark this answers is not in the document]
Comment (FSXAC) [reply src/thunderbots/software/util/visualizer_messenger/visualizer_messenger.h]: Oops, nice catch

File: src/thunderbots/software/util/visualizer_messenger/visualizer_messenger.h (hunk 1 above)
[the remark this answers is not in the document]
Comment (FSXAC) [reply src/thunderbots/software/util/visualizer_messenger/visualizer_messenger.h]: Updated

**Review by garethellis0 — APPROVED** [review]
LGTM :+1: :tada:

**Pull request merged**